ansible_builder/steps.py

```diff
@@ -37,16 +37,20 @@ class GalaxyInstallSteps(Steps):
     def __init__(self, requirements_naming):
         """Assumes given requirements file name has been placed in the build context
         """
+        relative_galaxy_path = os.path.join(
+            constants.user_content_subfolder, requirements_naming)
         self.steps = []
         self.steps.append(
-            "ADD {0} /build/{0}".format(requirements_naming)
+            "ADD {0} /build/{0}".format(
+                constants.user_content_subfolder)
```

Comment:
to me, this is a little funky. It ends up being:
```
ADD _build /build/_build
```

I think I would rather see it become:
```
ADD _build /build
```
or
```
ADD _build /tmp/_build
```
I'm leaning towards /tmp, given these really are temp files.

Reply:
> ADD _build /build/_build

Yes, what I expected.

> I'm leaning towards /tmp, given these really are temp files.

If they're temporary files, are the deleted in the final image? Or is our disposition that we want to keep them so that user can debug problems?

Reply:
They will never get into the final image, but it is more what the requirements.yaml file looks like for offline collections.

Today it would be
```
- name: /build/_build/ansible-netcommon-1.4.1.tar.gz
- name: /build/_build/foo-1-1-1.tar.gz
```